File: practice/800/263A-beautiful-matrix.py
for i in range(5):
    row = input().split()  # read each row
    for j in range(5):
        if row[j] == '1':  # found the 1
            print(abs(i - 2) + abs(j - 2))  # distance to centre (row 3, col 3 = index 2,2)

File: practice/800/263A-beautiful-matrix.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
    for (int i = 0; i < 5; i++)        // loop through rows
        for (int j = 0; j < 5; j++) {  // loop through columns
            int x; cin >> x;            // read each value
            if (x == 1)                 // found the 1
                cout << abs(i - 2) + abs(j - 2); // distance to centre (index 2,2)
        }
}